.github/workflows/docker-build-push.yml

```diff
@@ -4,7 +4,7 @@ name: Build, Render, and Push
 # events only for the master branch
 on:
   push:
-    branches: [ master ]
+    branches: [ staging, master, cansavvy/gha-updates-pr-process ]
```

Comment:
```suggestion
    branches: [ staging, master ]
```

Reply:
I did this in 7baef02

Reply:
As I said, irrelevant, but I couldn't delete it!

Reply:
Oh weird.